### ISSUE TITLE
Add pagination to /all radio page

### ISSUE THREAD

**Issue by hugurp** [internet4000/radio4000#126]
I think it is now time to add pagination to the page with all radios.

It takes too much time for this page to load and it therefore feels bloated.

The most complex aspect will be to make it work with the channel filtering (alphabetic, last updated and creation date).

I suggest a classic pagination at the bottom of the page, not an infinite scrolling that you cannot link to.
- first page & last page buttons (shallow query the db to have a length)
- three next and three previous pages (if there are)
- "go to page" input/select? (optional)

**Comment by oskarrough** [comment]
Yes! I think the more complex issues (search and sorting) you mention are necessary. But when we have that, pagination will be super sweet since everything will be fast!!
